libs/barista-components/overlay/src/overlay-trigger.ts

```diff
@@ -134,11 +134,15 @@ export class DtOverlayTrigger<T>
       enterEvent.stopPropagation();
       this._moveSub.unsubscribe();
       this._moveSub = this._ngZone.runOutsideAngular(() =>
-        fromEvent(this.elementRef.nativeElement, 'mousemove').subscribe(
-          (ev: MouseEvent) => {
-            this._handleMouseMove(ev, enterEvent);
-          },
-        ),
+        merge(
+          of(enterEvent).pipe(
+            delay(0),
+            map(() => enterEvent),
```

Comment:
I don't think this line is necessary.
```suggestion
```

Reply:
You're right, it was a typo since I had timer() as base observable before, so I needed to map it to the enterEvent. Thanks!